Source/WebCore/crypto/CryptoAlgorithmIdentifier.h

```diff
@@ -30,8 +30,7 @@
 namespace WebCore {
 
 enum class CryptoAlgorithmIdentifier {
-    RSASSA_PKCS1_v1_5 = 1,
-    RSA_PSS,
+    RSA_PSS = 1,
```

Comment:
Do these identifiers go into some persistent serialization? We probably cannot reuse numbers if so.